metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_report.py

```diff
@@ -16,7 +17,7 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class BigQuerySchemaApiPerfReport:
+class BigQuerySchemaApiPerfReport(Report):
     list_projects = PerfTimer()
     list_datasets = PerfTimer()
     get_columns_for_dataset = PerfTimer()
```

Comment:
```suggestion
    list_projects: PerfTimer = field(default_factory=PerfTimer)
    list_datasets: PerfTimer = field(default_factory=PerfTimer)
    get_columns_for_dataset: PerfTimer = field(default_factory=PerfTimer)
```

Not able to add suggestions for all here, but this is required for all fields in this and next report.